files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -860,7 +860,34 @@ protected function debug( $message ) {
 			return;
 		}
 
-		// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
-		error_log( sprintf( '%s (%s)', $message, wp_debug_backtrace_summary() ) );
+		$trace = $this->backtrace_fmt();
+
+		\Automattic\VIP\Logstash\log2logstash(
```

Comment:
Do we need the full function namespace to do the call?

Reply:
@pauarge yup, it's not used anywhere else in the file.